### DISCUSSION TITLE
OpenAPI/swagger export

### DISCUSSION THREAD

**Discussion by gaufranc** [thunderclient/thunder-client-support#962] [Ideas]
Hi there! 
Amazing extension. Been using for a while, thanks so much!

I have a question, is there a possibility that in the future we could export our collections as openapi/swagger format/schema? so we can build docs automatically, etc?

Thanks a lot again and have a good weekend.

**Comment by rangav** [comment]
@gaufranc good suggestion, please create a new feature request will add to roadmap.

**Reply by gaufranc** [reply]
Thanks a lot
